src/semidiscretization.jl

```diff
@@ -63,6 +63,7 @@ that can be passed to `solve` from the [SciML ecosystem](https://diffeq.sciml.ai
 """
 function semidiscretize(semi::AbstractSemidiscretization, tspan)
   u0_ode = compute_coefficients(first(tspan), semi)
+  mpi_isparallel() && MPI.Barrier(mpi_comm())
```

Comment:
Do we really need such a barrier, @sloede? I just copied it from Taam but wasn't sure whether you introduced it there to only synchronize the output report.
https://github.com/trixi-framework/Trixi.jl/blob/2031729176e38d3c157bd126641b4610dceb0be3/src/run.jl#L139

The same comment applies to the second `MPI.Barrier` below.

Reply:
Yeah, I think it would be good to open an issue for it" #328.

Reply:
OK, thanks! In that case I think you can remove it (or comment it out with a TODO?), as I don't know if this will be resolved before we kill Taam.

Reply:
Done in 314fca8.